data/json/recipes/recipes.json

```diff
@@ -5,7 +5,8 @@
     "//": "This is just a dummy category. No recipes should be added in here.",
     "recipe_subcategories": [
       "CSC_*_FAVORITE",
-      "CSC_*_RECENT"
+      "CSC_*_RECENT",
+	  "CSC_*_HIDDEN"
```

Comment:
```suggestion
	  "CSC_*_HIDDEN"
```

Reply:
Huh. Really not sure how that one happened -- those lines were evenly indented in my local copy.